### ISSUE TITLE
Pictures are not displayed in Edition mode

### ISSUE THREAD

**Issue by seafoox** [algolia/documentation-editor#24]
![image](https://cloud.githubusercontent.com/assets/1101220/13004445/cf150b54-d17b-11e5-993b-8c66c8e399f9.png)

I inspected the generated HTML, and it looks like the image url is not rendered in the template.

```
<img ng-src="{​{ section.content.images[0].image[0] }}" src="{​{ section.content.images[0].image[0] }}" data-pin-nopin="true">
```

**Comment by redox** [comment]
Actually, after looking at the page it seems it's rather a `{{` escaping issue (XSS protection). All angular attributes are wrongly escaped (cc @alexandremeunier).

![screen shot 2016-02-16 at 13 48 07](https://cloud.githubusercontent.com/assets/29529/13076666/10b9feb2-d4b4-11e5-874f-43d1860466fd.png)

**Comment by redox** [comment]
Fixed in d1bc2db417f0a5e4af3bd576074262e1edf16faa
